tools/python/aura_snapshot_voting/manual_voting/manual_votes.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Take a look at the protocol fee allocator.  All inputs and outputs shoudl be named by date or something, and you should figure they will collect over time.